src/components/VmDetails/itemListStyle.css

```diff
@@ -39,6 +39,8 @@
   flex-direction: row;
   flex-wrap: wrap;
   align-items: baseline;
+
+  max-width: 100%;
```

Comment:
This would also effect the Disks card layout - please verify it works ok.

Reply:
Will remove. I accidentally saved this file on VS Code after working on #1028

Reply:
Still need to do this...

Reply:
I realized this actually wasn't related to #1028. Right now, any item-row-info is a flex box without max-width, and it will overflow without this set limit. 
![screenshot-localhost-3000-2019 06 18-21-01-46](https://user-images.githubusercontent.com/25762021/59729425-85966680-920c-11e9-90a1-9cd8a3d425d1.png)